modules/end-user-guide/partials/proc_creating-workspaces.adoc

```diff
@@ -2,4 +2,223 @@
+        cpuLimit: 500m
+        cpuRequest: 30m
+        image: 'quay.io/che-incubator/che-code:insiders'
+        memoryLimit: 128Mi
+        memoryRequest: 32Mi
+        sourceMapping: /projects
+        volumeMounts:
+          - name: checode
+      name: vscode-injector
+    - attributes:
+        app.kubernetes.io/component: vscode-runtime
+        app.kubernetes.io/part-of: vscode.eclipse.org
+        controller.devfile.io/container-contribution: true
+      container:
+        cpuRequest: 30m
+        command:
+          - /checode/entrypoint-volume.sh
+        env:
+          - name: CODE_HOST
+            value: 0.0.0.0
+        memoryRequest: 256Mi
+        sourceMapping: /projects
+        cpuLimit: 500m
+        volumeMounts:
+          - name: checode
+            path: /checode
+        memoryLimit: 1024Mi
+        image: 'quay.io/che-incubator/che-code:insiders'
+        endpoints:
+          - attributes:
+              cookiesAuthEnabled: true
+              discoverable: false
+              type: main
+              urlRewriteSupported: true
+            exposure: public
+            name: che-code
+            protocol: https
+            secure: false
+            targetPort: 3100
+          - attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            exposure: public
+            name: code-redirect-1
+            protocol: http
+            targetPort: 13131
+          - attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            exposure: public
+            name: code-redirect-2
+            protocol: http
+            targetPort: 13132
+          - attributes:
+              discoverable: false
+              urlRewriteSupported: true
+            exposure: public
+            name: code-redirect-3
+            protocol: http
+            targetPort: 13133
+      name: vscode-runtime-description
+    - name: checode
+      volume:
+        ephemeral: true
+  events:
+    preStart:
+      - init-container-command
+----
+<1> Name of the `DevWorkspaceTemplate` custom resource. This name will be used to reference the `DevWorkspaceTemplate` custom resource within the `DevWorkspace` custom resource.
+<2> User namespace, which is the target {orch-namespace} for the new workspace.
+====
+
+. To prepare the `DevWorkspace` custom resource, copy the contents of the target Git repository's devfile.
++
+.Copied devfile contents with `schemaVersion: 2.1.0`
+====
+[source,yaml,subs="+quotes,+attributes"]
+----
+components:
+  - name: tooling-container
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-latest
+----
+====
++
+TIP: For more details, see the link:https://devfile.io/docs/2.1.0/what-is-a-devfile[devfile v2 documentation].
+
+. Create a `DevWorkspace` custom resource definition, pasting the devfile contents from the previous step under the `spec.template` field.
++
+.A `DevWorkspace` custom resource definition
```

Comment:
```suggestion
.A `DevWorkspace` custom resource
```

Reply:
@l0rd, I'm a bit confused. In your previous suggestion above, you suggest removing `definition` and in this suggestion you suggest adding `definition`. That's intended by you, correct? (Just double-checking.)

Reply:
I am removing the word `definition` in both comments.

Reply:
Apologies, looks like I got dyslexic while looking at the GItHub diffs.